.github/workflows/flutter_integration_test.yml

```diff
@@ -20,22 +20,56 @@ jobs:
   test-android:
     runs-on: macos-latest
     timeout-minutes: 30
+    defaults:
+     run:
+       working-directory: ./flutter/example
+    strategy:
+     fail-fast: false
+     matrix:
+       sdk: ['stable', 'beta']
+    steps:
+     - name: checkout
+       uses: actions/checkout@v3
+
+     - uses: actions/setup-java@v3
+       with:
+         distribution: 'adopt'
+         java-version: '11'
+
+     - uses: subosito/flutter-action@48cafc24713cca54bbe03cdc3a423187d413aafa # pin@v2.10.0
+       with:
+         channel: ${{ matrix.sdk }}
+
+     - name: flutter upgrade
+       run: flutter upgrade
+
+     - name: flutter pub get
+       run: flutter pub get
+
+     - name: launch android emulator & run android integration test
+       uses: reactivecircus/android-emulator-runner@50986b1464923454c95e261820bc626f38490ec0 #pin@v2.27.0
+       with:
+         working-directory: ./flutter/example
+         api-level: 21
+         arch: x86_64
+         profile: Nexus 6
+         script: flutter test integration_test/integration_test.dart --verbose
```

Comment:
We can apply those changes after merged here https://github.com/getsentry/sentry-dart/pull/1543
This helps with speeding up and flakiness.